docs/kubernetes/pouch_cri_annotations_supported.md

```diff
@@ -2,15 +2,18 @@
 
 * [Overview](#overview "Overview")
 * [The Annotations Supported](#the-annotations-supported "The Annotations Supported")
-  * [Make runtime choosing supported](#make-runtime-choosing-supported "Make runtime choosing supported")
-  * [Make lxcfs configurable supported](#make-lxcfs-configurable-supported "Make lxcfs configurable supported")
-* [Pull Request](#pull-request "Pull Request")
+  * [Runtime choosing](#runtime-choosing "Runtime choosing")
```

Comment:
Can we use RuntimeClass in Kubernetes to support runv/Kata which is described in https://github.com/alibaba/pouch/issues/2314? @Starnop @zhuangqh

Reply:
We use annotations to support runtime choosing for legacy kubernetes. RuntimeHandler of kubernetes 1.12 is also supported

Reply:
If that, I think we should add detailed instructions in this document. 
< 1.12, suggest using annotation;
>= 1.12, use runtime class

In addition, we should have a plan whether to deprecate the annotation for future.